google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -2279,8 +2279,7 @@ class SpannerInstanceIterator(resource_iter_class_factory(
 class StorageBucketIterator(resource_iter_class_factory(
         api_method_name='iter_storage_buckets',
         resource_name='storage_bucket',
-        api_method_arg_key='projectNumber',
-        resource_validation_method_name='storage_api_enabled')):
+        api_method_arg_key='projectNumber')):
```

Comment:
Can we instead update the storage_api_enabled() method to something along the line of

storage_enabled() := true if storage api or CAI api enabled, false otherwise?

By removing this I am afraid users might get a lot of 403 if CAI is not enabled

Reply:
I included thoughts around this in https://github.com/forseti-security/forseti-security/issues/2897#issuecomment-501501497 actually. 

In summary, my opinion is that we dont need to check any api in this StorageBucketIterator function because:
- This api validation in this function is done on the target project being scanned. (if we need CAI, we only need it on the Forseti project)
- This function eventually calls iter_cai_assets which works on data already in cai_temporary_store table and does not make any API calls